api/ui/src/main/kotlin/com/recco/api/ui/MainViewModel.kt

```diff
@@ -40,6 +43,9 @@ internal class MainViewModel @Inject constructor(
         initialLoadOrRetry()
     }
 
+    val sdkConfig: ReccoConfig
+        get() = appRepository.getSDKConfig()
```

Comment:
maybe we should fetch it within `initialLoadOrRetry()` and add `sdkConfig` as part of the `MainUI` or update `user.reccoStyle`